modules/FindHarfBuzz.cmake

```diff
@@ -61,3 +62,5 @@ if(NOT TARGET HarfBuzz::HarfBuzz)
         IMPORTED_LOCATION ${HARFBUZZ_LIBRARY}
         INTERFACE_INCLUDE_DIRECTORIES ${HARFBUZZ_INCLUDE_DIR})
 endif()
+
+set(HARFBUZZ_LIBRARIES "${HARFBUZZ_LIBRARY}")
```

Comment:
Just the following should be enough I think:
```cmake
set(HARFBUZZ_LIBRARIES ${HARFBUZZ_LIBRARY})
```

And that's also my last comment, then I'm happy to merge :)